CoordImporter/Parser/Parsers.cs

```diff
@@ -59,56 +62,51 @@ public static void LogParse(string tracker, string inputLine, GroupCollection gr
     Result<MarkData, string> Parse(string inputLine);
 }
 
-public class SirenParser : ITrackerParser
+public partial class SirenParser : ITrackerParser
 {
-    private static readonly char LinkChar = SeIconChar.LinkMarker.ToIconChar();
     private static readonly char I1Char = SeIconChar.Instance1.ToIconChar();
-    private static readonly char I9Char = SeIconChar.Instance9.ToIconChar();
     
     // For the format "(Maybe: Storsie) {LinkChar}Labyrinthos{I1Char} ( 17  , 9.6 ) "
-    private static readonly Regex SirenRegex = new Regex(
-        $@"(\(Maybe: (?<mark_name>[\w+ '-]+)\) {LinkChar})?(?<map_name>[\w+ '-]+)(?<instance>[{I1Char}-{I9Char}]?)\s+\(\s*(?<x_coord>[0-9\.]+)\s*,\s*(?<y_coord>[0-9\.]+)\s*\)",
-        RegexOptions.Compiled);
+    [GeneratedRegex($@"\(Maybe: (?<mark_name>[\w+ '-]+)\) \ue0bb(?<map_name>[\w+ '-]+)(?<instance>[\ue0b1-\ue0b9]?)\s+\(\s*(?<x_coord>[0-9\.]+)\s*,\s*(?<y_coord>[0-9\.]+)\s*\)", RegexOptions.Compiled)]
```

Comment:
Reading up on this attribute has lead me to see that `RegexOptions.Compiled` is unnecessary now, so we can axe it.

Source https://learn.microsoft.com/en-us/dotnet/standard/base-types/regular-expression-source-generators#source-generation

Reply:
Though one thing that really irks me is that they go through all this trouble of basically making `const` in c#, but you have to make it a method for it to work.

Like, MSFT pls, _you write the language_, make some syntactic sugar so you don't need to treat it as an Immediately Invoked Lambda 🤦